<ui version="4.0" >
 <class>BatchIDClass</class>
 <widget class="QDialog" name="BatchIDClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>BatchID</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>561</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title" >
    <string>BatchID File</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QLineEdit" name="lineEdit" />
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonBrowse" >
      <property name="text" >
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>220</y>
     <width>561</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title" >
    <string>ID Range</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" >
    <item>
     <widget class="QSpinBox" name="spinBox" >
      <property name="minimum" >
       <number>10060001</number>
      </property>
      <property name="maximum" >
       <number>10069999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_2" >
      <property name="minimum" >
       <number>10060001</number>
      </property>
      <property name="maximum" >
       <number>10069999</number>
      </property>
      <property name="value" >
       <number>10069999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton" >
      <property name="text" >
       <string>Batch</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>150</y>
     <width>561</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title" >
    <string>Output Directory</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" >
    <item>
     <widget class="QLineEdit" name="lineEdit_4" />
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonBrowse_2" >
      <property name="text" >
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>561</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title" >
    <string>Generate BatchID File</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" >
    <item>
     <widget class="QLineEdit" name="lineEdit_5" />
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonBrowse_3" >
      <property name="text" >
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2" >
      <property name="text" >
       <string>Generate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
